lightning/mydump/parquet_parser.go

```diff
@@ -19,6 +21,10 @@ import (
 
 const (
 	batchReadRowSize = 32
+
+	// if a parquet if small than this threshold, parquet will load the whole file in a byte slice to
+	// optimize the read performance
+	smallParquetFileThreshold = 256 * 1024 * 1024
```

Comment:
not sure this value should be configurated, since `region-concurrency` defaultly assigned number of CPU cores, the memory consumption is comparably less than CPU consumption